packages/vite/src/node/config.ts

```diff
@@ -710,7 +710,7 @@ export async function resolveConfig(
   await Promise.all(userPlugins.map((p) => p.configResolved?.(resolved)))
 
   if (process.env.DEBUG) {
-    debug(`using resolved config: %O`, {
+    debug(`using resolved config: 0%`, {
```

Comment:
I think this is intentional to format as object https://developer.mozilla.org/en-US/docs/Web/API/console#examples